### PR TITLE
Fixes/gantt fixes

### PR DESCRIPTION
**Pull request by joneff**
Fixes https://github.com/telerik/kendo-theme-bootstrap/issues/97
Fixes https://github.com/telerik/kendo-theme-default/issues/304
Fixes https://github.com/telerik/kendo-theme-default/issues/302
Fixes https://github.com/telerik/kendo-theme-default/issues/271

Note: somewhat related to https://github.com/telerik/kendo-theme-default/pull/297

### CODE REVIEW

**Comment by inikolova** [comment]
In my opinion, metrics should be kept in the _layout file

File: scss/gantt/_layout.scss [hunk 1]
```diff
@@ -196,7 +196,7 @@
     }
     .k-task-wrap {
         margin: 0 -1.5em;
-        padding: 0 25px;
+        padding: 5px 25px;
```
Comment (inikolova) [inline scss/gantt/_layout.scss:196]: @joneff , should this padding be exactly 5 / 25px. Is it related to some metrics? Can we use variables?

**Comment by joneff** [comment]
@inikolova  I agree. Still, in this particular case, we have different calculation of a grid header height. For instance in theme-default, grid header height is the sum of the button height plus the vertical cell padding, where as in theme-bootstrap it's as high as any other row.

**Review by inikolova — APPROVED** [review]

File: scss/gantt/_layout.scss (hunk 1 above)
Comment (joneff) [reply scss/gantt/_layout.scss:196]: It's related to the size of `k-task-draghandle`. and thus far, we don't have such variable...

**Pull request merged**